legion/engine/rendering/pipeline/default/defaultpipeline.cpp

```diff
@@ -41,16 +43,19 @@ namespace legion::rendering
 
         buffer modelMatrixBuffer;
         buffer entityIdBuffer;
+        buffer flipbookBuffer;
 
         {
             app::context_guard guard(context);
             addFramebuffer("main");
             modelMatrixBuffer = buffer(GL_ARRAY_BUFFER, sizeof(math::mat4) * 1024, nullptr, GL_DYNAMIC_DRAW);
             entityIdBuffer = buffer(GL_ARRAY_BUFFER, sizeof(id_type) * 1024, nullptr, GL_DYNAMIC_DRAW);
+            flipbookBuffer = buffer(GL_ARRAY_BUFFER, sizeof(size_type) * 1024, nullptr, GL_DYNAMIC_DRAW);
```

Comment:
uhm wait frameIds are size_type? but we use uints on the GPU? might wanna change these to uints

Reply:
ah yes, lmao, that does make sense